wpilib/wpilib/robotbase.py

```diff
@@ -37,98 +36,85 @@ class RobotBase:
     """
 
     def __init__(self):
-        # TODO: StartCAPI()
-        # TODO: See if the next line is necessary
-        # Resource.RestartProgram()
-        inst = NetworkTablesInstance.getDefault()
-        inst.setNetworkIdentity("Robot")
+        NetworkTables.setNetworkIdentity("Robot")
         
-        if not RobotBase.isSimulation():
-            inst.startServer("/home/lvuser/networktables.ini")
+        if self.isReal():
+            NetworkTables.startServer("/home/lvuser/networktables.ini")
         else:
-            inst.startServer()
+            NetworkTables.startServer()
         
         from .driverstation import DriverStation
         self.ds = DriverStation.getInstance()
 
-        inst.getTable("LiveWindow").getSubTable(".status").getEntry("LW Enabled").setBoolean(False)
+        NetworkTables.getTable("LiveWindow").getSubTable(".status").getEntry("LW Enabled").setBoolean(False)
         from .livewindow import LiveWindow
         LiveWindow.setEnabled(False)
 
         self.__initialized = True
 
     def free(self):
         """Free the resources for a RobotBase class."""
-        # TODO: delete?
         pass
 
     @staticmethod
-    def isSimulation():
-        """
-            :returns: If the robot is running in simulation.
-            :rtype: bool
+    def isSimulation() -> bool:
```

Comment:
are we gonna start adding type hinting to stuff?

Reply:
There is an issue that states we should do it. Adding it gradually is fine.

Reply:
Per #157, yes.

Whether we'll depend on the `typing` module whilst we still support Python 3.4 is an open question however.  But that's a discussion for over there.

Reply:
What if we made a rash decision to not support 3.4 in 2018?